packages/typescript/angular.js

```diff
@@ -6,21 +6,36 @@ module.exports = {
     "codelyzer"
   ],
   "rules": {
-    "component-class-suffix": true,
-    // "component-selector": [true, "element", "my", "kebab-case"],
-    "directive-class-suffix": true,
-    // "directive-selector": [true, "attribute", "my", "camelCase"],
     "import-destructuring-spacing": true,
     "no-attribute-parameter-decorator": true,
     "no-forward-ref": true,
-    "no-input-rename": true,
-    "no-output-rename": true,
     "no-shadowed-variable": true,
-    // "pipe-naming": [true, "camelCase", "my"],
-    "use-host-property-decorator": true,
+    "directive-selector": [
+      true,
+      "attribute",
+      "app",
+      "camelCase"
+    ],
+    "component-selector": [
+      true,
+      "element",
+      "app",
+      "kebab-case"
+    ],
     "use-input-property-decorator": true,
-    "use-life-cycle-interface": true,
     "use-output-property-decorator": true,
+    "use-host-property-decorator": true,
+    "no-input-rename": true,
```

Comment:
there are cases we deliberately rename inputs due to conflicting names with inner variables and we also want to keep a more clear name for input. in my opinion this rule is strict